test/infcover.c

```diff
@@ -293,6 +288,11 @@ static void inf(char *hex, char *what, unsigned step, int win, unsigned len, int
     mem_setup(&strm);
     strm.avail_in = 0;
     strm.next_in = NULL;
+
+    mem_limit(&strm, 1);
+    ret = PREFIX(inflateInit2)(&strm, win);
+                                                assert(ret == Z_MEM_ERROR);
```

Comment:
Bad indentation.

Reply:
This is how a lot of the tests in `infcover.c` are.

Reply:
Yeah... But only if the first part of line is past the tab position of assert() function... Otherwise it's all on same line.

Reply:
Ok it is fixed.